pkg/storkctl/clusterpair.go

```diff
@@ -258,18 +267,136 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				util.CheckErr(err)
 				return
 			}
-			printMsg("Using PX-Service Endpoint of DR cluster to create clusterpair...\n", ioStreams.Out)
-			ip, port, token, err := getClusterPairParams(dFile, dIP, dPort)
+
+			if len(sIP) > 0 || sPort != "9001" || len(dIP) > 0 || dPort != "9001" {
```

Comment:
Why return if port is not equal to 9001

Reply:
the default value for `src-port` and `dest-port` is 9001, hence even if these variables are not specified default value 9001 gets assigned to the vars sIP and dIP. If it is ok, we can remove the default value for these, so that we can just bail out for any of the values provided.

Reply:
Modified the default value of 9001 and added the check if any value is provided then to bail out.